sympy/solvers/ode.py

```diff
@@ -6477,8 +6477,9 @@ def _linear_2eq_order1_type1(x, y, t, r):
     """
     l = Symbol('l')
```

Comment:
This is not your fault but does this code work if a person has `l` (ell) instead of eps as her constant?  Should this line be `l = Dummy()`?